cibuildwheel/__main__.py

```diff
@@ -309,5 +309,16 @@ def detect_warnings(platform: str, build_options: BuildOptions) -> List[str]:
     return warnings
 
 
+def detect_setup_pyproject_existence(package_dir: str) -> bool:
+    exists = False
+    if os.path.exists(os.path.join(package_dir, 'setup.py')):
```

Comment:
You could also just do:
```python
    setup_py = os.path.exists(os.path.join(package_dir, "setup.py"))
    pyproject = os.path.exists(os.path.join(package_dir, "setup.py"))
    return setup_py or pyproject
```

Reply:
I recommend adding a check for setup.cfg too; setuptools now has a config-only mode.

```python
    return any(os.path.exists(os.path.join(package_dir, name)) 
        for name in ["setup.py", "setup.cfg", "pyproject.toml'])
```

Reply:
Interesting... didn't know about that. Replacing all content with that